less/icons/icons-png-fallback.css

```diff
@@ -222,6 +222,8 @@
 
 .fuelux-icon-multimail, .glyphicon-multimail { background-image: url('png/multimail.png'); background-repeat: no-repeat; .fuelux-icon; } 
 
+.fuelux-icon-new-window, .glyphicon-new-window { background-image: url('png/new-window.png'); background-repeat: no-repeat; .fuelux-icon; } 
```

Comment:
These appear to be gitignored. PNG usage is not prescribed and is a side effect of Grunt Icon believe. We probably should change the setup of Grunt Icon to not produce these.

https://github.com/ExactTarget/fuelux-mctheme/blob/master/.gitignore#L20